packages/graph-explorer/src/connector/gremlin/queries/createSubgraph.ts

```diff
@@ -46,38 +46,51 @@ const subgraphResult = async (
     req: SubGraphRequest,
     rawIds: Map<string, "string" | "number">
 ): Promise<SubGraphResponse> => {
+    let vertices: SubGraphResponse["vertices"] = []
+    let edges: SubGraphResponse["edges"] = []
 
-    
-    const vSG = subgraphTemplate({...req});
-    const eSG = subedgeTemplate({...req});
-    let [vData, eData] = await Promise.all([
-        gremlinFetch<RawSubVertRequest>(vSG),
-        gremlinFetch<RawSubEdgeRequest>(eSG)
-    ]);
-    console.log("VDATA AND EDATA")
-    console.log(vData.result.data["@value"])
-    console.log(eData.result.data["@value"])
+    if (req.canE.length <= 0 && req.canE.length <=0){
+      return { vertices, edges}
+    }
 
+    // Create the Vertices Result
+    const vSG = subgraphTemplate({...req});
+    const eSG = subedgeTemplate({...req})
```

Comment:
is this variable being used?

Reply:
why is it a duplicate of the line above?

Reply:
vSG = vertex sub graph query
eSG = edge sub graph query

Reply:
Oh, I see the difference now